core/domain/story_domain.py

```diff
@@ -662,11 +662,12 @@ def get_acquired_skill_ids_for_node_ids(self, node_ids):
 
         Args:
             node_ids: list(str). The list of IDs of the nodes inside
-            the story.
+                the story.
 
         Returns:
-            list(str). The list of acquired skill ids for the node ids.
-            If no aquired skill id is linked to them, the list will be empty.
+            list(str). The union of the acquired skill IDs corresponding to
+                each of the node IDs. If no acquired skill id is linked to
+                them, the list will be empty.
```

Comment:
If you want to keep the second sentence, please change "them" to "any of them" for clarity.

Reply:
Hmm but if only one skill has no acquired skill id, but others do, the list won't be empty. I understand that you might mean part of the union, but it may cause confusions.

Reply:
Oh I see, you're right. Let's just remove the second sentence then?

Reply:
Ok I will remove it :)

Reply:
Done.